appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockTab.java

```diff
@@ -18,6 +20,7 @@
   public static final String PROPERTY_NAME_TEXT = "Text";
```

Comment:
There should be no space prepending in the line. It is the same for the rest of code.